backend/review/views.py

```diff
@@ -82,26 +95,12 @@ def extra_metrics_section_filters_pcr(current_semester=None):
     """
     if current_semester is None:
         current_semester = get_current_semester()
-    return (
-        extra_metrics_section_filters
-        & Q(course__primary_listing_id=F("course_id"))
-        & Q(course__semester__lt=current_semester)
-        & ~Q(status="X")
-    )
-
+    return extra_metrics_section_filters & Q(course__semester__lt=current_semester) & ~Q(status="X")
```

Comment:
Why remove this filter? Couldnt' that lead to crosslisted courses being double counted?

Reply:
The idea is now crosslisted courses will be in separate topics so we don't want to count those out